ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/HardFork/Combinator/Forging.hs

```diff
@@ -15,8 +15,11 @@ module Ouroboros.Consensus.HardFork.Combinator.Forging (
   , hardForkBlockForging
   ) where
 
+import           Control.Monad (void)
```

Comment:
I _think_ this will be a redundant import in some versions. We usually use `import qualified Control.Monad as Monad` and use `Monad.void`. This is until we drop GHC 8.10 I think.

Reply:
Could you confirm this somehow? I've never heard of this before, and I'd hate to make the code more verbose for no good reason.

Reply:
Omit my comment for now and let's see what CI says. If we can avoid this change, even better.